UATagPreferenceCenter/UATagPreferenceCenter/Classes/UATagPreferencesViewController.swift

```diff
@@ -33,6 +33,11 @@ open class UATagPreferencesViewController: UIViewController, UITableViewDelegate
     
     public var preferenceDeviceData: [(tagPreference: UATagPreference, hasTag: Bool)]?
     
+    /**
+     * (optional) prefer to use preferences that have been updated remotely (defaults to true)
+     */
+    public var preferSavedPreferences: Bool = true
```

Comment:
New api? You should version it 1.1.0

Reply:
One of the features of the pref center is it's ability to have prefs updated through a push notification. However, you may not want to allow this to happen, which is where preferSavedPreferences comes in. It's already available in UATagPreferenceCenter:

```
public class func start(preferences: [UATagPreference], style: UATagPreferencesStyle = UATagPreferencesStyle(), title: String = "", preferSavedPreferences: Bool = true) {
```

but I forgot to include this if you're embedding the preference center. Would this be a version 1.0.1?